packages/vanilla/demo/index.ts

```diff
@@ -17,7 +17,14 @@ for (const [sel, items] of boxes) {
 
 const selection = new SelectionArea({
     selectables: ['body > section > div'],
-    boundaries: ['body > section']
+    boundaries: ['body > section'],
+    behaviour: {
```

Comment:
The demo page should only be a bare minimum and not contain any additional configuration ^^